doc/setup-binderhub.rst

```diff
@@ -378,3 +383,32 @@ Then update ``secret.yaml`` with the following::
 This token will be used for accessing the GitLab API, and is also used as the
 git password when cloning repos. With this token, no username is required to
 clone a repo.
+
+.. _dind:
+
+Use Docker-inside-Docker (DinD)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, BinderHub will build pods with the host Docker installation.
+This often means you are stuck with whatever version of Docker provided by your
+cloud provider. BinderHub supports an alternative that uses `Docker-in-Docker
+(DinD) <https://hub.docker.com/_/docker>`_. To turn `dind` on, you'll need to set
+the following configuration in your ``config.yaml`` file::
+
+    dind:
+      enabled: true
+      daemonset:
+      image:
```

Comment:
this and the next two lines need to be indented. Though they are the same as the default so we probably don't need them at all?

Reply:
I've fixed the indentation. I think it is helpful to show the default the image specification here though as this is the sort of thing someone may want to customize.